Cargo.toml

```diff
@@ -34,10 +34,12 @@ categories = ["command-line-utilities", "development-tools"]
 
 [dependencies]
 aho-corasick = "1.0.2"
+async-dup = "1.2.4"
```

Comment:
From the `smol` ecosystem. Provides `Arc` and `Mutex` variants that implement `AsyncWrite`.